### DISCUSSION TITLE
[Suggestion] for homes

### DISCUSSION THREAD

**Discussion by yetshadow** [Zrips/CMI#1293] [Feature Requests]
**Description of issue or feature request:**

---

Suggestion; Add the ability to invite players to home, you can look to uhome, this plugin do that and I'm pretty sure players will likes it.

https://www.spigotmc.org/resources/uhome-with-uuid-support.8805/

**Comment by Zrips** [comment]
Hows that different from /tpahere ?

**Comment by yetshadow** [comment]
You don't need to be logged in to have someone invited to your home, its perferct for mob spawners and towns.
Btw, i'm using your plugin since it was only intended as a check my inventory plugin, I'm very impressed with all the features you've added so far! I never tought that I could replace essentials.

**Comment by mrfloris** [comment]
Just allow players to set a warp to achieve this. /home is more of a personal/private thing, while /warp is more public.

**Comment by yetshadow** [comment]
Warps are public, this way, players can invite any player they want with /home invite playername homename

Like, if a player want to have their friends to come at his house to build something or whatever else reasons.

Per example, you made a big mob spawner, your friends do no need you to be online to tp at your spawner if you invite them to do so. And no, i don't want to add unlimited gomes to players.

**Comment by mercurialmusic** [comment]
I like the idea, but I wonder how it would work -- would that home just be added to the invited player's list? Seems like otherwise they'd be able to exploit it for extra sethomes. We just use Residence for this sort of thing.
